fastlane/Fastfile

```diff
@@ -0,0 +1,133 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+
+  before_all do
+    skip_docs
+    import_shared_fastfile
+  end
+
+  ######################### PUBLIC LANES ##########################
+
+  desc "Release a new version of the ios-sdk. Uploading a new release to a GitHub and CocoaPods trunk"
```

Comment:
`Release a new version of the ios-sdk` text here is probably wrong

Reply:
Thanks and updated in 1e4ec1c.